lib/tasks/cleanup.rake

```diff
@@ -57,6 +57,13 @@ namespace :cleanup do
     end
   end
 
+  desc 'Reindex banned users'
+  task :reindex_banned_users => :environment do
+    User.where("ban_text <> ''").find_each do |user|
+      user.xapian_mark_needs_index
+    end
+  end
+
```

Comment:
WDTK has 13094 banned users, so this might be too much to cope with in one run, but I think we should give it a go – I assume we just might get a few exceptions as the transaction fails from a locked record in a different process?